sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -81,3 +81,37 @@ def test_np_log():
         FunctionTransformer(np.log1p).transform(X),
         np.log1p(X),
     )
+
+
+def test_kw_arg():
+    X = np.arange(10).reshape((5, 2))
+
+    F = FunctionTransformer(np.around, kw_args=dict(decimals=3))
```

Comment:
rounding integers doesn't do anything, so this doesn't check anything, right? Unless you use negative decimals?